runtime/world.js

```diff
@@ -5,11 +5,9 @@
  * Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
  furnished to do so, subject to the following conditions: The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
  */
-const fs = require('fs-extra');
-const merge = require('merge');
-const requireDir = require('require-dir');
+const { setDefaultTimeout, Before } = require('@cucumber/cucumber');
 const getRemote = require('./getRemote');
-// const helpers = require('./helpers');
+const { filterQuietTags } = require('.././cucumber.js');
```

Comment:
This should be either require('../../cucumber.js'); or require('../cucumber.js');

Reply:
Done its '../'